tests/integration-e2e/test_loadbalance_integration.py

```diff
@@ -0,0 +1,89 @@
+"""Tests load balance."""
+# Since a good deal of load balance tests are e2e we put all of them together in this group
+import numpy as np
+import numpy.testing as npt
+import pytest
+from pathlib import Path
+
+SIM_DIR = Path(__file__).parent.parent.absolute() / "simulations"
+
+
+def _read_complexity_file(base_dir, pattern, cx_pattern):
+    import glob
+    # Construct the full pattern path
+    full_pattern = Path(base_dir) / pattern / cx_pattern
+
+    # Use glob to find files that match the pattern
+    matching_files = glob.glob(str(full_pattern))
```

Comment:
note that Path has a `.glob()`: https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob could save you an import and `str`

Reply:
yes, I will change that.